Quentinchampenois/decidim-module_phone_authorization_handler
# Feat: add address to proposal serializer

🎩 Description
Add proposal address in proposal serializer so the proposal address is available in admin_export of proposals.

📌 Related Issues
[Notion card](https://www.notion.so/opensourcepolitics/bd4bf860eab94f9daca7da803f09e3b2?v=c96df65755dd403d95e86f82d7749709&p=6ffef71f266e4dd8aa5adc1c541fdcd3&pm=c)

Testing
1. As an admin, go to a process
2. Go to proposals and export them, for example as excel
3. See that you have the address column in the export file
